app/viewers/embed/viewer/geo.rb

```diff
@@ -40,6 +40,13 @@ def index_map?
         index_map.present?
       end
 
+      # Returns true or false whether the viewer should display the Download All
+      # link. True for all geo objects, unless there is only one file
+      # (because the file download link will suffice for that)
+      def display_download_all?
+        @purl_object.downloadable_files.length > 1
+      end
```

Comment:
honestly, this seems like it could even be the default behavior in `common_viewer.rb` and then nothing to override here?